tools/jdbc/src/main/java/org/duckdb/DuckDBResultSet.java

```diff
@@ -1086,27 +1086,54 @@ public Array getArray(String columnLabel) throws SQLException {
 	}
 
 	public Date getDate(int columnIndex, Calendar cal) throws SQLException {
-		throw new SQLFeatureNotSupportedException("getDate");
+	    return getDate(columnIndex);
 	}
 
 	public Date getDate(String columnLabel, Calendar cal) throws SQLException {
-		throw new SQLFeatureNotSupportedException("getDate");
+	    return getDate(findColumn(columnLabel), cal);
 	}
 
 	public Time getTime(int columnIndex, Calendar cal) throws SQLException {
-		throw new SQLFeatureNotSupportedException("getTime");
+	    return getTime(columnIndex);
 	}
 
 	public Time getTime(String columnLabel, Calendar cal) throws SQLException {
-		throw new SQLFeatureNotSupportedException("getTime");
+	    return getTime(findColumn(columnLabel), cal);
 	}
 
 	public Timestamp getTimestamp(int columnIndex, Calendar cal) throws SQLException {
```

Comment:
Nope. And I'd love to see an example of it being used. But the descriptions I read were that it wanted an instant-based result and the `Calendar` is what the driver is currently using. Since we use the driver calendar to convert our instants to naïve timestamps in the non-`Calendar` version, this flavour doesn't have to do much.